package.json

```diff
@@ -37,9 +37,7 @@
     "babel-preset-stage-0": "^6.0.15",
     "bootstrap-loader": "^2.0.0-beta.21",
     "bootstrap-sass": "^3.3.7",
-    "butter-base-theme": "^0.1.2",
-    "butter-theme-dark": "^0.1.3",
-    "butter-theme-pink": "^0.0.1",
+    "butter-installer-theme": "^0.1.0",
```

Comment:
Well is not a theme either, not sure what you mean by that, but feel free to send a PR to make it an `"installer"`:

https://github.com/butterthemes/butter-installer-themes